src/app/add_entry/page.jsx

```diff
@@ -0,0 +1,157 @@
+"use client"
+import Link from 'next/link'
+import { supabase } from '../lib/supabase'
+import { useRouter } from 'next/navigation'
+import { UploadButton } from '../utils/uploadthing'
+import { useState } from 'react'
+
+const weatherOptions = [
+  "sunny",
+  "raining",
+  "cloudy"
+]
```

Comment:
would move this to a `siteDiaryConstants.js` file or similar. This could be used in many places (theoretically) so should be in a nice shareable location :)